Resources/Prototypes/Roles/Antags/nukeops.yml

```
@@ -79,13 +79,14 @@
 #Nuclear Operative Medic Gear
 - type: startingGear
   id: SyndicateOperativeMedicFull
-  parent: SyndicateOperativeGearFull
+  parent: SyndicateOperativeGearFullNoUplink
   equipment:
     eyes: ClothingEyesHudSyndicateAgent
     #outerClothing: ClothingOuterHardsuitSyndieMedic # goobstation - duke nukies
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)